auditors/capabilities/fixtures/capabilities-some-allowed-mix-old-labels.yml

```diff
@@ -0,0 +1,63 @@
+# this is to test backwards compatibility with old unregistered annotations (kubernetes.io)
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deployment
+  namespace: capabilities-some-allowed-multi-containers-mix-labels
+spec:
+  selector:
+    matchLabels:
+      name: deployment
+  template:
+    metadata:
+      labels:
+        name: deployment
+        audit.kubernetes.io/pod.allow-capability-chown: "SomeReason"
```

Comment:
tests pod and container override labels for the old annotation, so we make sure it's all still working